test/integration-test.py

```diff
@@ -356,16 +356,19 @@ def main():
         submissionStr = resp.read()
         if int(submissionStr) > 0:
             expected_ct_submissions = int(submissionStr)+1
+        next_tests = ""
+        if os.environ.get("BOULDER_CONFIG_DIR", "") == "test/config-next":
```

Comment:
It seems like you could just do this check in `test.js` directly, right? Seems simpler, especially since the check will be deleted once this migrates to the regular config.